### PR TITLE
feat: support setting affinity node labels

### PR DESCRIPTION
**Pull request by dingshun-cmss**
- Support setting affinity node labels in kblib.affinity

Fix: #620

### CODE REVIEW

File: addons/kblib/values.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ terminationPolicy: Delete
 ##
 availabilityPolicy: node
 
+## @param affinityNodeLabels
+##
+affinityNodeLabels: {}
```
Comment (shanshanying) [inline addons/kblib/values.yaml:16]: affinityNodeLabels is expected to be a map. could you pls give some more exmaples to show how to set the values.

**Review by ldming — APPROVED** [review]

File: addons/kblib/values.yaml (hunk 1 above)
Comment (dingshun-cmss) [reply addons/kblib/values.yaml:25]: Fixed in f8e85c0cb0314da89fc9739599eb412d00145aef

**Pull request merged**